Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -3430,125 +3569,160 @@ def __init__(self, dbot_score, name=None, entry_id=None, size=None, md5=None, sh
-                     publications=None, threat_types=None, behaviors=None, relationships=None):
+                     publications=None, threat_types=None, behaviors=None, relationships=None,
+                     creation_date=None, description=None, hashes=None, stix_id=None):
 
-            self.name = name
-            self.entry_id = entry_id
-            self.size = size
             self.md5 = md5
+            self.imphash = imphash
             self.sha1 = sha1
             self.sha256 = sha256
             self.sha512 = sha512
             self.ssdeep = ssdeep
+            self.hashes = hashes
+
+            self.associated_file_names = associated_file_names
+            self.community_notes = community_notes
+            self.creation_date = creation_date
+            self.description = description
             self.extension = extension
             self.file_type = file_type
-            self.hostname = hostname
             self.path = path
+            self.quarantined = quarantined
+            self.size = size
+            self.stix_id = stix_id
+            self.tags = tags
+            self.traffic_light_protocol = traffic_light_protocol
+
+            self.name = name
+            self.entry_id = entry_id
+            self.hostname = hostname
+
             self.company = company
             self.product_name = product_name
             self.digital_signature__publisher = digital_signature__publisher
             self.signature = signature
             self.actor = actor
-            self.tags = tags
+            self.organization = organization
+
             self.feed_related_indicators = feed_related_indicators
             self.malware_family = malware_family
             self.campaign = campaign
-            self.traffic_light_protocol = traffic_light_protocol
-            self.community_notes = community_notes
             self.publications = publications
             self.threat_types = threat_types
-            self.imphash = imphash
-            self.quarantined = quarantined
-            self.organization = organization
-            self.associated_file_names = associated_file_names
             self.behaviors = behaviors
             self.relationships = relationships
 
             self.dbot_score = dbot_score
 
         def to_context(self):
-            file_context = {}
+            file_context = {'Hashes': []}  # type: dict
 
             if self.name:
                 file_context['Name'] = self.name
+
+            if self.hashes:
+                file_context['Hashes'] = self.create_context_table(self.hashes)
+
             if self.entry_id:
                 file_context['EntryID'] = self.entry_id
+
             if self.size:
                 file_context['Size'] = self.size
+
             if self.md5:
                 file_context['MD5'] = self.md5
+                file_context['Hashes'].append({'type': 'MD5',
```

Comment:
Relevant for all the following places.
```suggestion
                file_context.setdefault('Hashes', []).append({'type': 'MD5',
```